CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # Release Notes
 
+## 0.3.1
```

Comment:
N.B.: I'm not releasing, just adding to release notes ahead of time. I'd like to wait for the brew tap release integration to land before releasing next to test that out.